packages/deployer/src/deploySolutionFromTemplate.ts

```diff
@@ -238,6 +242,7 @@ export function _deploySolutionFromTemplate(
             // why are we updating this same property vs adding a new one? seems confusing
             /* istanbul ignore else */
             if (itemId) {
+              itemTemplate.originalItemId = itemTemplate.itemId;
```

Comment:
This is something we can likely remove longer-term, but I found it useful for debugging so I'd like to keep it around at least until we refactor this function